pyccel/ast/mathext.py

```diff
@@ -2,17 +2,20 @@
 # This file is part of Pyccel which is released under MIT License. See the LICENSE file or #
 # go to https://github.com/pyccel/pyccel/blob/master/LICENSE for full license details.     #
 #------------------------------------------------------------------------------------------#
+""" Module containing objects from the math module understood by pyccel
+"""
 
 import math
 
-from pyccel.ast.variable  import Constant
-from pyccel.ast.internals import PyccelInternalFunction
+from pyccel.ast.core      import PyccelFunctionDef, Module
 from pyccel.ast.datatypes import (NativeInteger, NativeBool, NativeFloat,
                                   default_precision)
+from pyccel.ast.internals import PyccelInternalFunction
+from pyccel.ast.variable  import Constant
 
 __all__ = (
+    'math_mod',
     'math_constants',
```

Comment:
Shouldn't we remove `math_constants` as well, given that they are contained in `math_mod` now?

Reply:
No, the fortran and c printers seem to use this dictionary to get the values for printing

Reply:
See [here](https://github.com/pyccel/pyccel/blob/master/pyccel/codegen/printing/ccode.py#L1748-L1759) and [here](https://github.com/pyccel/pyccel/blob/master/pyccel/codegen/printing/fcode.py#L159)

Reply:
I see. That raises another question though: why don't we just print the Python value of those constants instead of raising a `NotImplementedError`?

Reply:
I think that is a question for a separate issue. But I agree, the value in the python constant is not necessarily less precise than the C equivalent (e.g. `M_PI`) and even if it is, then I think it is better to print something than to refuse completely

Reply:
Could you please create an issue for the constants? It sounds like a good first issue for new contributors.

Reply:
See #1047

Reply:
Thanks!